<img src="./unal.png" align=left width="150" height="150"></img>

$\;$
---


<h2 align="center">PRUEBA DE MEDIA (para n grande, $\sigma^2$ desconocido)
    <h3 align="center">"prueba aproximada"

    
### Oscar Julian Layton 
---
    
$\;$



Para realizar una prueba de de media considerese el siguiente teorema para **n grande** (sin importar distribución):

## El teorema del límite central

<div class="alert alert-success">

Sea $\{ X_1, X_2, X_3,...,X_n \}$ una muestra aleatoria de una distribución **X** con media $\mu$ y varianza $\sigma^2$, para un **n** lo suficientemente grande, _$\overline{X}$_ se distribuye _**aproximadamente**_ normal con media $\mu$ y varianza $\frac{\sigma^2}{n}$, e.o.p: 

$$\overline{X} \underset{n\rightarrow \infty }{\sim} N(\mu, \sigma^2 /n) \;\;\;\;\;\; \;\;\;\;\;\; ; \;\;\;\;\;\; \;\;\;\;\; \frac{\overline{X}- \mu}{\sqrt{\sigma_x^2/n}} \sim N(0,1)$$

Tengase en cuenta que al decir muestra aleatoria se afirma que la sucecion de variables aleatorias son independientes igualmente distribuidas.
</div>


#### 1. Parámetro Poblacional

El parámetro poblacional a contrastar es $\mu$

#### 2. Hipótesis

Considérese la varianza poblacional $\sigma^2$


\begin{equation}
H_{0}: \mu = \mu_o \hspace{0.9cm} vs \hspace{0.9cm} H_{1}: \mu  < \mu_o
\end{equation}

#### 3. Estadística de prueba

Supóngase que $x_1, x_2, ,..., x_n$ es una muestra aleatoria de **X**, entonces por el TCL (Teorema Central del Límite):

\begin{equation}
\frac{\overline{X}_n- \mu}{\sqrt{\sigma_x^2/n}}\hspace{0.4cm} \xrightarrow[n\rightarrow \infty ]{D}\hspace{0.4cm} Z_{c} \hspace{1.2cm} ;  \hspace{1.2cm} Z_{c} \mid _{H_0} \sim N(0,1)
\end{equation}

#### 4. Construcción regla de desición

Para un $0<\alpha<1$ nivel de significancia

$$P_{\mu_{o}}(Error\; tipo \; I)=\alpha$$
$$P_{\mu_{o}}(Rechaz\;H_o\;cuando \;es \;verdadera)=\alpha$$
$$P_{\mu_{o}}(Z_c < Z_{(1-\alpha)} \; cuando \; Z_c \sim N(0,1))=\alpha$$

En consecuencia:

$$P_{\mu_{o}}\left (   \frac{\overline{X}_n- \mu}{\sqrt{\sigma_x^2/n}}< Z_{1-\alpha} \right )=\alpha$$

#### 5. Regla de Desición


<div class="alert alert-danger">

Para un $\alpha$ de significancia se procede a:

\begin{equation}
\vartheta :  \; \;Rechazar \; H_o  \;\;\;\; sii \;\; \;\; Z_{c} < Z_{(1-\alpha)}
\end{equation}
</div>

---


Para la **potencia de la prueba** se usa el error tipo II

$$\beta(\mu')=P(Error\; tipo \; II) $$

$$\beta(\mu')= P(Z_c > Z_{(1-\alpha)} \; cuando \; \mu =\mu^{´})$$


$$\beta(\mu') = P(\overline{X}_n > \mu_o + Z_{(1-\alpha)}(\sigma^2/\sqrt{n}) \;\; con \;\; \mu =\mu^{´})$$

$$\beta(\mu') =  \left (   \frac{\overline{X}_n- \mu^{´}}{\sigma/\sqrt{n}} > Z_{(1-\alpha)} + \frac{\mu_o- \mu^{´}}{\sigma/\sqrt{n} } \right )$$
$$potencia = 1 - \beta$$

---
## Ejemplo

Considerese el datset **births** del paquete openintro el cual contiene información sobre 150 nacimientos junto con información de las madres. Se quiere determinar si el peso de los niños es menor que 9kg.


In [11]:
library(openintro)
library(tidyverse)
data(births)
head(births, 4)

f_age m_age weeks premature visits gained weight sex_baby smoke    
1 31    30    39    full term 13      1     6.88   male     smoker   
2 34    36    39    full term  5     35     7.69   male     nonsmoker
3 36    35    40    full term 12     29     8.88   male     nonsmoker
4 41    40    40    full term 13     30     9.00   female   nonsmoker

In [14]:
data<-births$weight
head(data, 4)

#### a) Hipótesis apropiadas sobre la media.

En este caso interesa hacer inferencia con respecto a $\mu$ peso promedio de la población (recien nacidos). En este orden de ideas la hipótesis a contrastar es:

\begin{equation}
H_{0}: \mu = 9 \hspace{0.9cm} vs \hspace{0.9cm} H_{1}: \mu < 9
\end{equation}

#### b) Uso de la estadística de prueba.

Tenemos entonces los siguientes datos: $n=150$, $\bar{x}=7.046$, se usa el estadístico de prueba: 

$$Z=\frac{\overline{x}-\mu}{s/\sqrt{n}}   \;\;\;\;\;\;; \;\;\;\;\;\; Z \;  \sim  \;  Z_{(1-\alpha)}$$

In [22]:
mean(data)
sd(data)

[1] 7.046

[1] 1.49721

In [23]:
mu<-9
Z <- (mean(data)-mu)*sqrt(length(data))/sd(data)  ; Z

[1] -15.98407

De esta forma, para la información disponible tenemos que: 

$$Z=\frac{\sqrt[]{n}(\bar{x}-\mu_{0})}{s}=\frac{\sqrt[]{150}(7.046-9)}{1.497}=-15.98$$

Luego se tiene:

In [26]:
########################################################################################################################
###########       NORMAL ESTANDAR       qnorm(p, mean = 0, sd = 1, lower.tail = TRUE, log.p = FALSE)   #################
########################################################################################################################

alpha<-0.05

#conociendo z necesario
Z_alpha <- qnorm(alpha, 0 , 1, lower.tail = TRUE) # lower.tail=TRUE ent P[X <= x]
Z_alpha

[1] -1.644854

$$  Z_{(1-\alpha)} = Z_{0.05} =-1.64$$

#### c) Desición

 $ \vartheta :$ Luego como $Z= -15.98$, se tiene que a un nivel de significancia de $ \alpha= 0.05 $, $\;\;$ $Z= -15.98 < Z_{(0.05)}=-1.64 $ con lo cual se concluye que se RECHAZA la hipótesis nula $H_o$ afirmandose que el peso promedio de los recien nacidos es mejor a 9kg. esto tambien se evidencia en el p_valor = 8.80729630768984e-58 (el cual es muy pequeño):

In [27]:
p_valor <- pnorm(-15.98, 0 , 1, lower.tail = TRUE); p_valor         # lower.tail=TRUE ent P[X <= x]

[1] 8.807296e-58

---
#### d) Prueba en R junto con intervalo de confianza

In [30]:
t.test(x = data,  alternative = "less",  mu = 9, conf.level  = 0.95)  
#alternative="greater" or "less" or two.sided


	One Sample t-test

data:  data
t = -15.984, df = 149, p-value < 2.2e-16
alternative hypothesis: true mean is less than 9
95 percent confidence interval:
     -Inf 7.248336
sample estimates:
mean of x 
    7.046 
